Documentation/Storage-Configuration/Block-Storage-RBD/block-storage.md

```diff
@@ -142,7 +143,8 @@ kubectl get svc wordpress
     ```console
     NAME        CLUSTER-IP   EXTERNAL-IP   PORT(S)        AGE
     wordpress   10.3.0.155   <pending>     80:30841/TCP   2m
-    ```
+
+```
```

Comment:
It may mess up the rendering if these ticks change indentation. This was an automatic change in your editor?

Reply:
yes, reverted